github/interactions.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+// InteractionsService handles communication with the repo and org related
+// methods of the GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/interactions/
+type InteractionsService service
+
+// Interaction represents the interaction restrictions
+// for repository and organisation
```

Comment:
For the sake of the godocs, please make all doc comments be complete sentences followed by a period.

Also, for consistency, let's please use the American English spellings, for example: `organization`.
If you are unsure, you can copy the spellings from the GitHub Developer API docs.

Additionally, I noticed you did a force push... please do not use those in PRs for this repo, as it makes things easier for reviewers to see only the changes since the last review.  Thank you!